app/assets/stylesheets/govuk_publishing_components/components/_phase-banner.scss

```diff
@@ -6,3 +6,18 @@
     margin-right: govuk-spacing(2);
   }
 }
+
+.gem-c-phase-banner--inverse {
+  background: $govuk-brand-colour;
+
+  .govuk-phase-banner__content__tag {
+    background: govuk-colour("white");
+    color: $govuk-brand-colour;
+  }
+
+  .govuk-phase-banner__content__app-name,
+  .govuk-phase-banner__text,
+  .govuk-phase-banner__text a {
```

Comment:
Can this `a` be an explicit class?

Reply:
Sure - I'll also change the examples and default to use the `govuk-link` class - but means that any message with a link in it passed through needs to use that class as well.